src/Compilers/CSharp/Portable/Binder/Binder.ValueChecks.cs

```diff
@@ -272,6 +272,11 @@ private BoundExpression CheckValue(BoundExpression expr, BindValueKind valueKind
             return ToBadExpression(expr, resultKind);
         }
 
+        public CompoundUseSiteInfo<AssemblySymbol> GetNewCompoundUseSiteInfo(BindingDiagnosticBag futureDestination)
```

Comment:
Nit: it doesn't feel like this is the right location for this function. Perhaps in `Binder.cs` instead, given that it's a general helper?

Reply:
I had the same reaction. Does this have anything to do with value checks?

---
In reply to: [369822539](https://github.com/dotnet/roslyn/pull/41117#discussion_r369822539) [](ancestors = 369822539)